src/clarity/datagrid/datagrid-column.ts

```diff
@@ -16,14 +16,18 @@ import {Sort} from "./providers/sort";
     selector: "clr-dg-column",
     template: `
         <!-- I'm really not happy with that select since it's not very scalable -->
-        <ng-content select="clr-dg-filter, clr-dg-string-filter"></ng-content>
-        
-        <clr-dg-string-filter *ngIf="field && !customFilter" 
-            [clrDgStringFilter]="defaultFieldFilter"></clr-dg-string-filter>
-            
-        <button class="datagrid-column-title" [disabled]="!sortable" (click)="sort()">
-            <ng-content></ng-content>
-        </button>
+        <div class="datagrid-column-flex">
```

Comment:
Adding `display:flex;` to `.datagrid-column` will break the flow. 

Indeed, flex can be added to `.datagrid-column` but that will require a total refactor for the whole datagrid component CSS (moving from `display: table-row;`  to `display: flex;`). If you guys are ok with such changes, I can start working on that.

@adityarb88 @mathisscott